src/lib/constants/selector.py

```diff
@@ -0,0 +1,2 @@
+class LoginPage:
```

Comment:
Hey, can you add (object) to all class definitions as per https://google-styleguide.googlecode.com/svn/trunk/pyguide.html?showone=Classes#Classes